s4e-web/src/app/views/map-view/state/overlay/overlay.model.ts

```diff
@@ -14,13 +15,25 @@ export interface Overlay {
   id: string;
   type: OverlayType;
   caption: string;
+  // :TODO: below fields have not been added to the API yet, they might need to be renamed later on
```

Comment:
This TODO isn't necessary. If someone will extend or modify it he will know where to find the model.